docs/pages/access-controls/sso/okta.mdx

```diff
@@ -3,6 +3,128 @@ title: Authentication With Okta as an SSO Provider
 description: How to configure Teleport access using Okta for SSO
 videoBanner: SM4Am-i8cj4
 ---
+# Getting Started
```

Comment:
There's already an H1 for this page (it's derived from the `title` frontmatter field).
```suggestion
```